server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -71,18 +88,106 @@ public void unregister(D dispatcher) {
     /**
      * Posts the message to the bus.
      *
-     * @param message the message to post
-     * @param responseObserver the observer to receive outcome of the operation
+     * <p>Use the {@code Bus} class abstract methods to modify the behavior of posting.
+     *
+     * @param message  the message to post
+     * @param observer the observer to receive outcome of the operation
+     * @see #post(Iterable, StreamObserver) for posing multiple messages at once
+     */
+    public final void post(T message, StreamObserver<IsSent> observer) {
+        checkNotNull(message);
+        checkNotNull(observer);
+        checkArgument(isNotDefault(message));
+
+        post(singleton(message), observer);
+    }
+
+    /**
+     * Posts the given messages to the bus.
+     *
+     * <p>Use the {@code Bus} class abstract methods to modify the behavior of posting.
+     *
+     * <p>The {@link StreamObserver} argument is the posting outcome observer;
+     * the {@link StreamObserver#onNext StreamObserver.onNext} will be called for each message
+     * passed to the bus.
+     *
+     * <p>A valid message will result in
+     * an {@link io.spine.base.Status.StatusCase#OK OK} status {@link IsSent} instance.
+     *
+     * <p>An invalid message will result in an {@link io.spine.base.Error Error} status
+     * {@link IsSent} instance.
+     *
+     * <p>Depending on the underlying {@link MessageDispatcher}, a message which causes a business
+     * {@link io.spine.base.Failure} may result ether a {@link io.spine.base.Failure} status or
+     * an {@link io.spine.base.Status.StatusCase#OK OK} status {@link IsSent} instance. Usually,
+     * the {@link io.spine.base.Failure} status may only pop up if the {@link MessageDispatcher}
+     * processes the message sequentially and throws
+     * the {@linkplain io.spine.base.FailureThrowable FailureThrowables} (wrapped in a
+     * {@link RuntimeException}) instead of handling them. Otherwise, the {@code OK} status should
+     * be expected.
+     *
+     * <p>Note that the {@code observer} is always positive, i.e.
```

Comment:
```
Note that {@linkplain StreamObserver#onError StreamObserver.onError()} is never called for the passed observer, since errors are propagated as statuses of {@code IsSent} response.
```